File: CPP/SayuranOrganik.cpp
#include <iostream>
#include <string>
#include "Sayuran.cpp" 

using namespace std;

class SayuranOrganik : public Sayuran {
private:
    string jenis_pupuk;
    string sertifikasi;
    int harga_per_kg;

public:
    SayuranOrganik(string id_t = "", string nama_t = "", string asal = "", 
                   string kat = "", int panen = 0, string gizi = "", 
                   string pupuk = "", string sertif = "", int harga = 0)
        : Sayuran(id_t, nama_t, asal, kat, panen, gizi) {
        this->jenis_pupuk = pupuk;
        this->sertifikasi = sertif;
        this->harga_per_kg = harga;
    }

    string get_jenis_pupuk() { return jenis_pupuk; }
    void set_jenis_pupuk(string pupuk) { this->jenis_pupuk = pupuk; }

    string get_sertifikasi() { return sertifikasi; }
    void set_sertifikasi(string sertif) { this->sertifikasi = sertif; }

    int get_harga_per_kg() { return harga_per_kg; }
    void set_harga_per_kg(int harga) { this->harga_per_kg = harga; }
};

File: PHP/SayuranOrganik.php
<?php
class SayuranOrganik {
    private $id;
    private $nama;
    private $asalDaerah;
    private $kategori;
    private $masaPanen;
    private $kandunganGizi;
    private $jenisPupuk;
    private $sertifikasi;
    private $hargaPerKg;
    private $gambar;

    public function __construct($id, $nama, $asalDaerah, $kategori, $masaPanen, $kandunganGizi, $jenisPupuk, $sertifikasi, $hargaPerKg, $gambar = '') {
        $this->id = $id;
        $this->nama = $nama;
        $this->asalDaerah = $asalDaerah;
        $this->kategori = $kategori;
        $this->masaPanen = (int)$masaPanen;
        $this->kandunganGizi = $kandunganGizi;
        $this->jenisPupuk = $jenisPupuk;
        $this->sertifikasi = $sertifikasi;
        $this->hargaPerKg = (int)$hargaPerKg;
        $this->gambar = !empty($gambar) ? $gambar : 'images/bayam.png';
    }

    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getAsalDaerah() { return $this->asalDaerah; }
    public function getKategori() { return $this->kategori; }
    public function getMasaPanen() { return $this->masaPanen; }
    public function getKandunganGizi() { return $this->kandunganGizi; }
    public function getJenisPupuk() { return $this->jenisPupuk; }
    public function getSertifikasi() { return $this->sertifikasi; }
    public function getHargaPerKg() { return $this->hargaPerKg; }
    public function getGambar() { return $this->gambar; }
}